docs/faq/faq.rst

```diff
@@ -112,3 +112,17 @@ This figure illustrates the relationship between different batch size configurat
 https://excalidraw.com/#json=pfhkRmiLm1jnnRli9VFhb,Ut4E8peALlgAUpr7E5pPCA
 
 .. image:: https://github.com/user-attachments/assets/16aebad1-0da6-4eb3-806d-54a74e712c2d
+
+How to generate ray timeline to analyse performance of a trainning job?
+------------------------------------------------------------------------------------------
+
+To generate the ray timeline file, you can set the config term ``ray_init.timeline_file`` to a json file path.
+For example:
+
+.. code:: bash
+
+    ray_init:
+      timeline_file: /tmp/ray_timeline.json
+  
+The file will be generated in the specified path at the end of a training job.
+You can use tools like chrome://tracing or the Perfetto UI and view the ray timeline file.
```

Comment:
could you add an example timeline figure here for illustration purpose so users know what results to expect? (please PR images to https://github.com/eric-haibin-lin/verl-community to keep repo size)